pkgs/development/compilers/binaryen/default.nix

```diff
@@ -2,13 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "binaryen";
-  version = "102";
+  version = "108";
 
   src = fetchFromGitHub {
     owner = "WebAssembly";
     repo = "binaryen";
     rev = "version_${version}";
-    sha256 = "sha256-UlktpY9tyjYNkmiBZM42QGg67kcPo7VDy2B4Ty1YIew=";
+    sha256 = "sha256-CcGxPBdUiNLfMjjJKFMdDvaIrHvf2M/gCub4JBw4+8c=";
+    fetchSubmodules = true;
```

Comment:
We should use our googletest instead.

Reply:
I don't think we have a googletest?

Reply:
Update: we have a gtest, but the package will require some patching to use this

Reply:
Added this and enabled tests.